SFS_binaural_synthesis/ir_generic.m

```diff
@@ -85,7 +85,9 @@
     % If needed interpolate the given impulse response set and weight, delay the
     % impulse for the correct distance
     ir = get_ir(sofa,X,[phi 0],x0(ii,1:3),'cartesian',conf);
-
+    % Correct Green's function amplitude
+    ir = ir/(4*pi);
```

Comment:
A unit power point Source and 3D free-field Green's function are both the same and are described by exp(-jkr)/(4pi*r). The 1/(4*pi) factor preserves unit sound power (see https://en.wikipedia.org/wiki/Sound_intensity#Inverse-square_law ). For HRTFs, the 1/(4*pi) is already included in the measurement (the measurements have not been done with a calibrated measurement system, though). The factor 1/(4*pi) should only be applied inside ``dummy_irs`` in order to get the correct free-field impulse response for a point source.